packages/lit-dev-tools-cjs/src/search/plugin.ts

```diff
@@ -29,6 +31,8 @@ type DocTypes =
   | DocType<'Tutorial', 'tutorial'>
   | DocType<'Docs', 'docs'>
   | DocType<'API', 'api'>
+  | DocType<'Video', 'video'>
+  | DocType<'MDN', 'other'>
```

Comment:
Similar comment as above. Should this also include WebKit, Stackblitz, and Web.dev?

Reply:
removed first generic parameter from DocType